### ISSUE TITLE
Feature Request: Use minimal resources if the window is minimized.

### ISSUE THREAD

**Issue by kursad-k** [o3de/o3de#17561]
**Is your feature request related to a problem? Please describe.**

The editor.exe uses at least %30 of my RTX GPU while it is minimized in the background on Windows. This is really unnecessary amount of power draw if it can be avoided

**Describe the solution you'd like**
O3DE should use less resources as much as possible if it is minimized. Godot engine is able to do this. It goes into almost no GPU use if it is in the background. 

**Describe alternatives you've considered**
The only alternative is to kill the process which is undesirable.

**Additional context**
I forgot to close O3DE at night couple times (while it was in the background), and it literally used %30-%35 GPU and utilized power all night.  The engine can be a bit more environment friendly in my opinion.

**Comment by lumberyard-employee-dm** [comment]
@kursad-k For more context, is the Editor open, but it is not in Game mode?
So the only thing being Editor rendered would be the viewport.

 I believe the rendering of the viewport isn't paused when the Editor doesn't have focus, so it is still using resources.
Someone from @o3de/sig-graphics-audio can correct me if I am wrong.

**Comment by kursad-k** [comment]
Yes I am specifically talking about the editor resource uses. The editor is not in game mode, just the idle sitting  editor  uses %20-35 of GPU in the background and I personally believe that this is really unnecessary uses of energy and resources. It would be very nice if this usage  is pushed down because the work around is to close O3DE everytime the user needs to change to some other app which is unrealistic in production work.
